### PR TITLE
fix: update grey to cloud color

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Update grey color to use updated grey color scheme.

## Changes

- Update grey colors to use correct grey color

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/182282552

### CODE REVIEW

File: src/components/DocumentUtility/DocumentUtility.tsx [hunk 1]
```diff
@@ -47,8 +47,9 @@ export const DocumentUtility: FunctionComponent<DocumentUtilityProps> = ({ docum
               }}
             >
               <ButtonIcon
-                className="bg-white border-2 border-cloud-100 rounded-xl hover:bg-gray-100"
+                className="bg-white border-2 border-cloud-100 rounded-xl hover:bg-cloud-100"
                 aria-label="document-utility-qr-button"
+                style={{ width: "auto", height: "auto" }}
```
Comment (simboonlong) [inline src/components/DocumentUtility/DocumentUtility.tsx:52]: why need auto?

File: src/components/UI/Tag/Tag.stories.tsx [hunk 2]
```diff
@@ -21,8 +21,8 @@ export const BorderedRed = () => {
   return <TagBordered className="text-red-500 border-red-500">TagBorderedRed</TagBordered>;
 };
 
-export const BorderedGraySmall = () => {
-  return <TagBorderedSm className="text-gray-600">BorderedGraySmall</TagBorderedSm>;
+export const BorderedGreySmall = () => {
```
Comment (simboonlong) [inline src/components/UI/Tag/Tag.stories.tsx]: retain previous name else `BorderedCloudSmall`?

File: src/components/DocumentUtility/DocumentUtility.tsx (hunk 1 above)
Comment (isaackps) [reply src/components/DocumentUtility/DocumentUtility.tsx:52]: there is a default value of 40px that will make it not center.

File: src/components/UI/Tag/Tag.stories.tsx (hunk 2 above)
Comment (isaackps) [reply src/components/UI/Tag/Tag.stories.tsx]: ok will update accordingly

File: src/components/DocumentUtility/DocumentUtility.tsx (hunk 1 above)
Comment (simboonlong) [reply src/components/DocumentUtility/DocumentUtility.tsx:52]: ic right. able to put `max-width: 100%` to `Printer` and `Download` instead? if cannot then nvm

Comment (isaackps) [reply src/components/DocumentUtility/DocumentUtility.tsx:52]: hmmm but it makes no diff leh

**Review by simboonlong — APPROVED** [review]

**Pull request merged**